monai/apps/auto3dseg/auto_runner.py

```diff
@@ -427,10 +427,10 @@ def set_gpu_customization(
                         }
                     }
 
-            ALGO: the name of algorithm. It could be one of algorithm names (e.g., 'dints') or 'unversal' which
+            ALGO: the name of algorithm. It could be one of algorithm names (e.g., 'dints') or 'universal' which
                 would apply changes to all algorithms. Possible options are
 
-                - {``"unversal"``, ``"dints"``, ``"segresnet"``, ``"segresnet2d"``, ``"swinunetr"``}.
+                - {``"universal"``, ``"dints"``, ``"segresnet"``, ``"segresnet2d"``, ``"swinunetr"``}.
```

Comment:
I think `segresnet` and `segresnet2d` do not support these anymore. Please help confirming @myron @dongyang0122 .